### PR TITLE
Upgraded project sdk version to 14.0.0, updated launch config template

### PR DESCRIPTION
**Pull request by JonasImprobable**

### CODE REVIEW

File: SpatialOS/launch_configs/deployment.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "template": "small",
+  "template": "w2_r0500_e5",
```
Comment (Helcaraxan) [inline SpatialOS/launch_configs/deployment.json]: Could you put this in a separate PR instead as it has nothing to do with 14.0 and should (normally) have been done a while ago. 🙂

**Comment by Helcaraxan** [comment]
To be entirely clear I suppose indeed that you have tested this to build before putting up the PR?

File: HelloWorker/src/CsharpWorker.csproj [hunk 2]
```diff
@@ -16,29 +16,29 @@
...
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.dll">
-      <Link>CoreSdkDll.dll</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.dll">
+      <Link>improbable_worker.dll</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.lib">
-      <Link>CoreSdkDll.lib</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.lib">
```
Comment (dgavedissian) [inline HelloWorker/src/CsharpWorker.csproj]: The `.lib` on Windows is actually not required at all for C# (as it's an DLL import library for C/C++). Therefore, you can just delete this.

File: client/src/Client.cs [hunk 3]
```diff
@@ -173,28 +171,20 @@ private static Connection ConnectClientReceptionist(string[] arguments)
...
                     logMessageBuilder.Append(
-                        String.Format("The EntityIdResponse ID value was {0}", response.Response.Value.Get().Value));
+                        String.Format("The EntityIdResponse ID value was {0}", response.Response.Value));
                 }
 
                 connection.SendLogMessage(LogLevel.Warn, LoggerName, logMessageBuilder.ToString());
             }
             else
             {
-                var workerType = response.Response.Value.Get().Value.workerType;
```
Comment (dgavedissian) [inline client/src/Client.cs]: :heart_eyes:

File: client/src/CsharpWorker.csproj [hunk 4]
```diff
@@ -26,20 +26,20 @@
...
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.dll">
-      <Link>CoreSdkDll.dll</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.dll">
+      <Link>improbable_worker.dll</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.lib">
-      <Link>CoreSdkDll.lib</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.lib">
```
Comment (dgavedissian) [inline client/src/CsharpWorker.csproj]: Same here

File: SpatialOS/launch_configs/deployment.json (hunk 1 above)
Comment (JonasImprobable) [reply SpatialOS/launch_configs/deployment.json]: Reverted the template change.

File: HelloWorker/src/CsharpWorker.csproj (hunk 2 above)
Comment (JonasImprobable) [reply HelloWorker/src/CsharpWorker.csproj]: Is that the case for Mac and Linux too?

Comment (JonasImprobable) [reply HelloWorker/src/CsharpWorker.csproj]: Removed the obsolete import. :)

File: client/src/CsharpWorker.csproj (hunk 4 above)
Comment (JonasImprobable) [reply client/src/CsharpWorker.csproj]: Removed.

File: SpatialOS/launch_configs/deployment.json (hunk 1 above)
Comment (JonasImprobable) [reply SpatialOS/launch_configs/deployment.json]: Ok, I fucked up. I accidentally merged the PR thinking that this was reverted. I even removed the changelog entry. Sorry about that.

Comment (Helcaraxan) [reply SpatialOS/launch_configs/deployment.json]: @JonasImprobable if you could please create another PR with the changelog that'll be alright. 🙂
